app-modules/form/src/Filament/Blocks/UploadFormFieldBlock.php

```diff
@@ -55,6 +55,9 @@ public function fields(): array
         return [];
     }
 
+    /**
+     * @return array<string, string>
```

Comment:
This return type is not fully correct as some of these values can be something other than `string` like `null` for example.